lib/harvest-notifier/templates/daily_report.rb

```diff
@@ -32,8 +66,12 @@ def generate # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
 
       private
 
-      def emails
-        assigns[:users].map { |u| u[:email] }.join(", ")
+      def current_date
```

Comment:
Could we pass date outside with `assigns[:date]` and format it with Slack https://api.slack.com/reference/surfaces/formatting#date-formatting ?